src/renderer/views/SettingsOverlay/index.tsx

```diff
@@ -108,7 +111,24 @@ async function handlePlayerUpdate() {
   return result;
 }
 
+const awsSinkGuid: string | undefined = ipcRenderer.sendSync(
+  "get-aws-sink-cloudwatch-guid"
+);
+const InfoTextStyled = styled("div", {
+ 
+  bottom: 50,
+  left: 50,
+  dragable: false,
+});
+
 function SettingsOverlay({ onClose, isOpen }: OverlayProps) {
+  let versionToDisplay = "unknown";
+  try {
+    versionToDisplay = CURRENT_VERSION;
+  } catch (error) {
```

Comment:
this `catch` clause seems to be a little bit weird since the `try` block has only one assigning statement. 😕 

can we elide try-catch blocks? (or replacing it with a`CURRENT_VERSION` check might be also good).

Reply:
Hmm... well, I thought this is relatively safer for "just in case", since pre-declared `CURRENT_VERSION` const is not nullable and build process itself would _probably_ break if that's the case, so basically it would filter in case build success and somehow working while fail to fetch proper version. 

Actually I might add some more other validation logic in that `try`. as far as I remember we have some version-switcheroo logic in internal target CI.